flow-node

```diff
@@ -84,6 +84,10 @@ if (options.length > 0) {
 
 require('./register')({ all: all });
 
+function withoutOptions(optionsToExclude, options) {
+  return options.filter((option) => !optionsToExclude.includes(option))
```

Comment:
`Array.includes()` requires Node v6 or above, is that okey or should we make this <= v6 compatible as well?

Reply:
I'd prefer to support older node versions. Can you please replace the `.includes()` call with something back-compatible like `indexOf() !== -1`?